packages/mesos-modules/extra/logsink.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Can you add this configuration to dcos-config-windows.yaml like we do in linux version
https://github.com/dcos/dcos/blob/8c6ec915d7653a0fd288148c391454097f3f4bf2/gen/dcos-config.yaml#L256-L278